pytables.ipynb

```diff
@@ -0,0 +1,287 @@
+{
```

Comment:
Add title markdown cell at beginning, so viewers know what the point of the notebook is